cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/ndnlive/Documents/NDNlive/.waf-1.7.15-81b0724e6c87060fe73ce80b83ec6372/waflib/Tools/c_config.py", line 458, in run_c_code\n    bld.compile()\n  File "/home/ndnlive/Documents/NDNlive/.waf-1.7.15-81b0724e6c87060fe73ce80b83ec6372/waflib/Build.py", line 188, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 140397249552784: cxxprogram test.cpp.1.o -> testprog}\n[\'/usr/bin/g++\', \'-Werror\', \'-stdlib=libc++\', \'test.cpp.1.o\', \'-o\', \'/home/ndnlive/Documents/NDNlive/build/.conf_check_7b1712bce7d387bb0a598634f015c484/testbuild/testprog\', \'-Wl,-Bstatic\', \'-Wl,-Bdynamic\']\n'
